pkg/build/strategies/sti/sti_test.go

```diff
@@ -59,6 +60,7 @@ func newFakeSTI(f *FakeSTI) *STI {
 		fs:        &test.FakeFileSystem{},
 		tar:       &test.FakeTar{},
 		preparer:  f,
+		ignorer: &ignore.DockerIgnorer{},
```

Comment:
Yes, I have a hook in emacs which runs gofmt whenever I save a Go file.  I'll re-open this file and try to explicitly run gofmt.